src/phoenix/server/api/helpers/playground_spans.py

```diff
@@ -264,6 +267,10 @@ def input_value_and_mime_type(
     yield INPUT_VALUE, safe_json_dumps(input_data)
 
 
+def prompt_id_metadata(prompt_id: Optional[GlobalID]) -> Iterator[tuple[str, Any]]:
+    yield METADATA, {"phoenix-prompt-id": str(prompt_id)}
```

Comment:
<img width="787" alt="image" src="https://github.com/user-attachments/assets/aa5b1755-8e34-4ec7-8d44-eef67f124a85" />

Do we always want to set this even when no prompt id is set?

Reply:
I feel like the name (e.g. the identifier is the better thing to add here)

Reply:
nit: I think snake case is easier to read in the tables

Reply:
that's a good point. we store the global ID in zustand but I don't think swapping it out is too big of a deal? Will need to update the gql schema to use Identifier instead of GlobalID too

Reply:
the backend can look this up too.

Reply:
I was able to pull it in easily and pass it through